content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -404,7 +406,7 @@ model CategoriesOnPosts {
 In this example, the `assignedAt` field stores additional information about the relation between `Post` and `Category`:
 
 - `assignedAt` stores _when_ the post was added to the category
```

Comment:
@janpio sorry, I've just realised I'd need to update this explanation too - if we switch to `createdAt`, should I remove this?

Reply:
Yep I think so. Might be easier to do in an unrelated PR to this one, and then close this when it is merged.